Rakefile

```diff
@@ -594,6 +687,15 @@ def header str
   puts ""
 end
 
+# Returns [gem_name, gem_version]
+def split_tag str
+  return [nil, str] if str == "master" # Support use of "master" even without gem name
+  fail "'tag' must be in the format <gem>/<version> Actual: #{str}" unless str.include?("/")
+  parts = str.split("/")
```

Comment:
The tag format is actually `"{PACKAGE}/v{VERSION}"`. We had a regex for this before, why isn't that being used here?